eos-full-node/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Default values for eos-node.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+initimage:
+  repository: santiment/eos
+  tag: latest
+  pullPolicy: IfNotPresent
+
+service:
+image:
+  repository: santiment/eos
+  tag: latest
+  pullPolicy: IfNotPresent
+
+service:
+  type: ClusterIP
+  port1: 8888
+  type: ClusterIP
+  port2: 9876
```

Comment:
These port1 and port2 variables are not used anywhere. I propose to add `externalPort` and `internalPort` port here for configuring the service and probably just remove the `9876` port from this file as it is used only for internal communication of the node.

Reply:
Good idea.  I'll fix that now.